manim/animation/creation.py

```diff
@@ -177,8 +236,13 @@ def update_submobject_list(self, index):
 
 
 class AddTextLetterByLetter(ShowIncreasingSubsets):
-    """
-    Add a Text Object letter by letter on the scene. Use time_per_char to change frequency of appearance of the letters.
+    """Show a :class:`~.Text` letter by letter on the scene.
```

Comment:
Is this really only for Test? or also for MathTex?

Reply:
I believe it only works for Text, but I might be wrong.

Reply:
Just tested it, it really *only* works for Text